processor/attributesprocessor/config.go

```diff
@@ -85,3 +110,31 @@ const (
 	//no action is performed.
 	DELETE Action = "delete"
 )
+
+// MatchProperties specifies the set of properties in a span to match against
+// and if the span should be included or excluded from the processor.
+// At least one of services or attributes must be specified. It is supported
+// to have both specified, but this requires all of the properties to match
+// for the inclusion/exclusion to occur.
+type MatchProperties struct {
+
+	// Services specify the list of service name to match against.
+	// A match occurs if the span service name is in this list.
+	// Note: This is an optional field.
+	Services []string `mapstructure:"services"`
+
+	// Attributes specifies the list of attributes to match against.
+	// All of these attributes must match exactly for a match to occur.
+	// Note: This is an optional field.
+	Attributes []Attribute `mapstructure:"attributes"`
+}
```

Comment:
Question: if both fields are optional what happens if both are missing? Does it match everything or it matches nothing? Or is it an invalid configuration?

Reply:
It will be an invalid configuration, ie if the field `include` or `exclude` is set then at least one of `attributes` or `services` must be specified with a non empty value. I will update the comments. 

ex invalid configurations
```
attributes/bad1:
  include:
  actions: ...

attributes/bad2:
  exclude:
    # Bad because services and attributes hvae the empty setting.
    services:
    attributes:
  actions: ...
```

ex of valid configuratoin

```
attributes/good:
  include:
    services: ["svca", "svcb"]
 actions: ...
```

Reply:
Sounds good. Perhaps worth adding a short comment here or in the config docs to explain that it is not valid.